mllib-dal/src/main/native/LinearRegressionDALImpl.cpp

```diff
@@ -15,16 +15,36 @@
  *******************************************************************************/
 
 #include <chrono>
+#include <unistd.h>
+
+#include <iomanip>
 #include <iostream>
 #include <vector>
 
+#ifdef CPU_GPU_PROFILE
+#include "GPU.h"
+#ifndef ONEDAL_DATA_PARALLEL
+#define ONEDAL_DATA_PARALLEL
+#endif
```

Comment:
compile error here due to some changes to the include logic, you need to include Common.hpp, pls follow KMeansImpl.cpp